di-ipv-cimit-stub/lambdas/post-mitigations/src/main/java/uk/gov/di/ipv/core/postmitigations/domain/PostMitigationsResponse.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.di.ipv.core.postmitigations.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+
+@Builder
+public record PostMitigationsResponse(@JsonProperty("result") String result) {}
```

Comment:
(you don't generally need `@JsonProperty` if it matches the param name, unless you think it helps clarity)